### PR TITLE
Fix PyGetInt for python 3

### PR DESCRIPTION
**Pull request by WilliamBruneau**
Before fix:

```
Python 3.7.2 (default, Jan 10 2019, 23:51:51) 
[GCC 8.2.1 20181127] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from miasm.analysis.machine import Machine
>>> machine = Machine("x86_64")
>>> jitter = machine.jitter("python")
>>> jitter.cpu.RAX = -1
OverflowError: can't convert negative int to unsigned

The above exception was the direct cause of the following exception:

SystemError: PyEval_EvalFrameEx returned a result with an error set

```

After fix:

```
Python 3.7.2 (default, Jan 10 2019, 23:51:51) 
[GCC 8.2.1 20181127] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from miasm.analysis.machine import Machine
>>> machine = Machine("x86_64")
>>> jitter = machine.jitter("python")
>>> jitter.cpu.RAX = -1
>>> hex(jitter.cpu.RAX)
'0xffffffffffffffff'
```

### CODE REVIEW

File: test/jitter/jitcore.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+jitter.cpu.EAX = -2
+assert jitter.cpu.EAX == 0xfffffffe
+
+jitter.cpu.RAX = -0xffffffffffffffff
+assert jitter.cpu.RAX == 1
+
+try:
+        jitter.cpu.RAX = 0x1ffffffffffffffff
+except TypeError as te:
```
Comment (p-l-) [inline test/jitter/jitcore.py]:
```suggestion
except TypeError:
```

File: test/jitter/jitcore.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        pass
+else:
+        raise Exception("Should see that 0x1ffffffffffffffff is too big for RAX")
+
+try:
+        jitter.cpu.RAX = 0x10000000000000000
+except TypeError as te:
```
Comment (p-l-) [inline test/jitter/jitcore.py]:
```suggestion
except TypeError:
```

File: test/jitter/jitcore.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+jitter.cpu.EAX = -0xefffffff
+assert jitter.cpu.EAX == 0x10000001
+
+jitter.cpu.EAX = -0xFFFFFFFF
+assert jitter.cpu.EAX == 1
+
+try:
+        jitter.cpu.EAX = -0x1ffffffff
+except TypeError as te:
```
Comment (p-l-) [inline test/jitter/jitcore.py]:
```suggestion
except TypeError:
```

**Comment by serpilliere** [comment]
Thank a lot @WilliamBruneau !

**Pull request merged**